### PR TITLE
Update cutest.rb

### PR DESCRIPTION
**Pull request by lovezx1990**
I don't quite understand ...

### CODE REVIEW

**Comment by dpo** [comment]
What don't you quite understand?

File: cutest.rb [hunk 1]
```diff
@@ -15,8 +15,7 @@ class Cutest < Formula
 
   depends_on "dpo/cutest/archdefs" => :build
   depends_on "dpo/cutest/sifdecode" => :build
-  depends_on "homebrew/versions/gcc43" => [:build, "enable-fortran"] if build.with? "matlab" # Mathworks only support gfortran 4.3.
-  depends_on :fortran
```
Comment (dpo) [inline cutest.rb]: We also need `:fortran` more generally, not just when `--with-matlab` is selected.

File: cutest.rb [hunk 2]
```diff
@@ -15,8 +15,7 @@ class Cutest < Formula
...
+  depends_on "homebrew/homwbrew-versions/gcc43" => [:build, "enable-fortran"] if build.with? "matlab"
```
Comment (dpo) [inline cutest.rb]: There's a typo here, but the fully-qualified path is erroneous anyways.

**Comment by lovezx1990** [comment]
I’m so sorry that I made the mistake. I did not mean to submit it, and I hope I am not disturbing you.

I try to install CUTEst with the Matlab interface, but I’m not familiar with everything……MAC OS X and GitHub. In fact, I got my Macbook just yesterday.

Now, I’m able to install it —without Matlab interface, as the lack of Yosemite support in gcc43.
However, it seems that gcc and gfortran 4.3 are the only ones supported by the Mathworks on OSX. I have tried the following method:


https://github.com/Homebrew/homebrew-versions/pull/662

which seems to be useful :

MBP:~ XZ$ brew install /Users/……/gcc43.rb 
Warning: gcc43-4.3.6 already installed


But I get the error as before：

MBP:~ XZ$ brew install cutest --HEAD --with-matlab
==> Installing cutest from dpo/homebrew-cutest
gcc43: OS X Mavericks or older is required.
Error: An unsatisfied requirement failed this build.

I'm in a puzzle as to what to do next.

**Comment by dpo** [comment]
@lovezx1990 Thanks for alerting me of that! I wonder how many Matlab users will get bitten.

You seem to already have gcc43 installed. Is that a Homebrew gcc43? If not, you can try to use it:
````
brew edit cutest
# Comment out the dependency on homebrew/versions/gcc43 in cutest.rb, save and close the editor
brew install cutest --with-matlab
````

Let me know if that works.

**Comment by lovezx1990** [comment]
Thanks a lot for your help. I have changed cutest.rb as you suggest:
` #depends_on "homebrew/versions/gcc43" => [:build, "enable-fortran"] if build.with? "matlab" # Mathworks only support gfortran 4.3. `


and then I try to install:



```
MBP:~ XZ$ brew install cutest --HEAD --with-matlab
==> Installing cutest from dpo/homebrew-cutest
==> Using Homebrew-provided fortran compiler.
This may be changed by setting the FC environment variable.
==> Building with an alternative Fortran compiler
This is unsupported.
==> Cloning http://ccpforge.cse.rl.ac.uk/svn/cutest/cutest/trunk
Updating /Library/Caches/Homebrew/cutest--svn-HEAD
==> ./install_cutest < cutest.input

 ERROR:  you need to install gfortran-4.3/gcc-4.3
 to allow for Matlab and CUTEst interaction. Aborting 

READ THIS: https://git.io/brew-troubleshooting
If reporting this issue please do so at (not Homebrew/homebrew):
  https://github.com/dpo/homebrew-cutest/issues
```

which seems like the same kind of problem. Is it because I did something wrong?

**Comment by dpo** [comment]
I can finally close this. Although the Mathworks no longer support gfortran, the most recent version of CUTEst contains a workaround so users can use the latest version of gfortran if they wish.

**Pull request closed without merging**